src/utils/generateArtifacts.ts

```diff
@@ -37,7 +37,7 @@ function decorateEnvrionmentsWithAppId(
   for (const [key, value] of Object.entries(environments)) {
     decoratedEnvrionment[key] = {
```

Comment:
Nitpick: `Environment` is misspelled.

Reply:
We remove this piece of logic on the plugin. But I noted this to fix it on the CLI.